packages/opencensus-propagation-jaeger/test/test-jaeger-format.ts

```diff
@@ -62,6 +62,46 @@ describe('JaegerPropagation', () => {
       );
       assert.deepStrictEqual(jaegerFormat.extract(getter), spanContext);
     });
+
+    it('should format traceId for 64Bit Hex id without leading zeros', () => {
+      const spanContext = jaegerFormat.generate();
+      spanContext.traceId = '70c2f20bd65603bd';
+      const getter = helperGetter(
+        `${spanContext.traceId}:${spanContext.spanId}::${spanContext.options}`
+      );
+      assert.deepStrictEqual(jaegerFormat.extract(getter), spanContext);
+    });
+
+    it('should format traceId for 64Bit Hex id with leading zeros', () => {
+      const spanContext = jaegerFormat.generate();
+      spanContext.traceId = 'c2f20bd65603bd';
+      const compareSpanContext = spanContext;
```

Comment:
You need to copy this, otherwise the line below is modifying the same object
```suggestion
      const compareSpanContext = {...spanContext};
```